spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -232,6 +232,8 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private volatile int defaultTxSize = DEFAULT_TX_SIZE;
 
+	protected volatile boolean defaultDurableSubscription = true;
+
```

Comment:
This would make things more in line with Kafka, where we'd need to start consumer groups at `earliest` by default, since the producer cannot safely reset the offset of a consumer.